### PR TITLE
[DC-1749] Update parameter names in registered tier script

### PR DESCRIPTION
**Pull request by ksdkalluri**

### CODE REVIEW

**Review by nishanthpp93 — COMMENTED** [review]
The changes look good. Is this already verified to run as expected?

**Comment by ksdkalluri** [comment]
> The changes look good. Is this already verified to run as expected?

I ran the script to check if it running by giving test names, I did not run the whole script to generate a new died dataset.

File: data_steward/tools/deid_runner.sh [hunk 1]
```diff
@@ -9,8 +9,8 @@ Usage: deid_runner.sh
   --cdr_id <combined_dataset name>
   --vocab_dataset <vocabulary dataset name>
   --dataset_release_tag <release tag for the CDR>
-  --cope_survey_dataset <dataset where RDR provided cope survey mapping table is loaded>
-  --cope_survey_table_name <name of the cope survey mappig table>
+  --cope_lookup_dataset_id <dataset where RDR provided cope survey mapping table is loaded>
+  --cope_table_name <name of the cope survey mappig table>
```
Comment (nishanthpp93) [inline data_steward/tools/deid_runner.sh:13]: typo: `mapping`

**Review by nishanthpp93 — COMMENTED** [review]
That sounds good! Verified that the parameter names matched the cleaning rule for cope_survey_info. Added a small suggestion for the spelling.

**Review by nishanthpp93 — APPROVED** [review]

**Pull request merged**